test/unit/eval_algorithms/test_common.py

```diff
@@ -91,15 +91,80 @@ def test_save_dataset(tmp_path, file_name):
             if json_obj[DatasetColumns.MODEL_INPUT.value.name] == "hello":
                 assert json_obj[DatasetColumns.CATEGORY.value.name] == "Age"
                 assert json_obj["scores"] == [
-                    {"name": "rouge", "value": 0.5, "error": None},
-                    {"name": "bert_score", "value": 0.42, "error": None},
+                    {"name": "rouge", "value": 0.5},
+                    {"name": "bert_score", "value": 0.42},
                 ]
 
             if json_obj[DatasetColumns.MODEL_INPUT.value.name] == "world":
                 assert json_obj[DatasetColumns.CATEGORY.value.name] == "Gender"
                 assert json_obj["scores"] == [
-                    {"name": "rouge", "value": 0.314, "error": None},
-                    {"name": "bert_score", "value": 0.271, "error": None},
+                    {"name": "rouge", "value": 0.314},
+                    {"name": "bert_score", "value": 0.271},
+                ]
+
+
+@pytest.mark.parametrize("file_name", ["my_dataset.jsonl", "my_dataset"])
```

Comment:
Can you please parametrize `ds_items` and the expected `json_obj` values so that you can combine this test with the previous test (`test_save_dataset`)?